.github/workflows/automerge.yml

```diff
@@ -18,6 +18,17 @@ jobs:
       - uses: julia-actions/setup-julia@082493e5c5d32c1fa68c35556429b0f1b2807453 # v1.0.1
         with:
           version: ${{ matrix.julia-version }}
+      - name: Cache artifacts
+        uses: actions/cache@v1
+        env:
+          cache-name: cache-artifacts
+        with:
+          path: ~/.julia/artifacts
+          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Registry.toml') }}
```

Comment:
@SaschaMann do you think is correct to use `Registry.toml` here?  Is my understanding correct that this would invalidate the cache every time this file changes, which would be quite often?

Reply:
Yes, that's correct. But it falls back to the `restore-keys` if this key doesn't match an existing cache, so it might still have some benefit even if this file changes quite a lot.

I'm not familiar with AutoMerge/RegistryCI. Do you want to cache artifacts used by their dependencies or some package-dependent artifacts that are used by the to-be-registered package?

`.ci/Manifest.toml` and `.ci/Project.toml` might be a better choice if this is about caching AutoMerge/RegistryCI dependencies.

Reply:
> Do you want to cache artifacts used by their dependencies or some package-dependent artifacts that are used by the to-be-registered package?

Ideally both, but mainly at least the dependencies of the AutoMerge pipeline

Reply:
> some package-dependent artifacts that are used by the to-be-registered package

We add the package being registered inside a temporary depot. The path of that temporary depot is created by `mktempdir`. So it is going to be tricky to cache that.

Reply:
I see, good point, so only dependencies of AutoMerge then.  I'm going to change this file to `.ci/Project.toml` then

Reply:
Done!